src/wallet/wallet.cpp

```diff
@@ -5320,6 +5320,12 @@ bool CWallet::CreateLelantusMintTransactions(
                         nValueToSelect = mintedValue + nFeeRet;
                     }
 
+                    if (!MoneyRange(mintedValue) || mintedValue == 0) {
+                        valueAndUTXO.erase(itr);
+                        skipIfNegative = true;
```

Comment:
bad variable name. It is known by this time the value is negative, no ifs. Really confusing